test/acceptance/features/steps/generic_testapp.py

```diff
@@ -113,3 +115,11 @@ def set_common_label(context, label):
 def check_env_var_value_in_both_apps(context, name, value):
     polling2.poll(lambda: context.application1.get_env_var_value(name) == value, step=5, timeout=400)
     polling2.poll(lambda: context.application2.get_env_var_value(name) == value, step=5, timeout=400)
+
+
+@step(u'The application "{app_name}" container has only one env "{envVar}" set')
```

Comment:
The step is a bit unclear, because container at runtime has or does not have an env var set. Here we speak about env vars declared in application resource. While here, would you also like to make the step where we do not need to specify application name, like:

```
The container declared in application resource contains env "{envVar}" set only once'
```

Reply:
+1